### ISSUE TITLE
Toolbar CSS Issue

### ISSUE THREAD

**Issue by michaelmarcuccio** [stephanrauh/ngx-extended-pdf-viewer#1279]
More CSS toolbar issues following the recent updates:

Toolbar Button/Icon alignment vertically is not matching. Presentation mode icon and Find icon are examples here. Both with different CSS issues. The Find has negative  top margin on the SVG, and the Presentation mode instead has no top margin on the button which makes both of these misaligned.

![image](https://user-images.githubusercontent.com/20432260/161343654-61036bcc-bde5-4caa-9c52-e623029494b1.png)

Version: 12.1.1

**Comment by stephanrauh** [comment]
Incredible. It seemed to be such a small change... truth to tell, I already suspected the CSS rules to be unnessecary convoluted (to avoid the word "broken") for a long time.

If you've got an idea how to simplify stuff, fire away. I've inherited the HTML code and the CSS code from the Mozilla team, and they simplified it some time ago, too. Of course, if I don't hear from you, I'll try to fix the bug, too.
